rastervision_pytorch_backend/rastervision/pytorch_backend/examples/tiny_spacenet.py

```diff
@@ -61,16 +58,19 @@ def make_scene(scene_id, image_uri, label_uri):
     # Use the PyTorch backend for the SemanticSegmentation pipeline.
     chip_sz = 300
     backend = PyTorchSemanticSegmentationConfig(
+        data=SemanticSegmentationGeoDataConfig(
+            scene_dataset=scene_dataset,
+            window_opts=GeoDataWindowConfig(
+                method=GeoDataWindowMethod.random,
+                size=chip_sz,
+                size_lims=(chip_sz, chip_sz + 1),
```

Comment:
What is `size_lims` and why is it `chip_sz + 1`?

Reply:
Fixed